docs/appendices/release-notes/5.7.2.rst

```diff
@@ -79,3 +79,6 @@ Fixes
 - Fixed an issue leading to a ``ArrayIndexOutOfBoundsException`` when using a
   correlated sub-query where some of its outputs weren't used in a parent
   query.
+
+- Fixed an issue leading to out-of-memory errors when executing ``lead`` or
+  ``lag`` window functions on a large table.
```

Comment:
No example as I don't have a concrete OOM reproduction. 

I saw map growing when running `select lag(x) over(order by x) as l from t` on a large table but it was tripped by CircuitBreaker. 

Should I rephrase and say "might lead..." instead of "leading to"   as we know that we clearly lacked accounting but no 100% reproducible scenario?

Reply:
+1 for `Fixed an issue that could lead to ...`

Reply:
rephrased